packages/server/src/service/trpc.ts

```diff
@@ -69,6 +69,22 @@ export class VbmRpc implements ImplRpc<IVbmRpc, Request> {
     if (!result) return error('Unable to find contact')
     return data(result)
   }
+  public supportEmail = async (
+    authorEmail: string, authorName: string, text: string,
+  ) => {
+    try {
+      return await mg.messages().send({
+        to: 'support@votebymail.io',
+        cc: authorEmail,
+        from: authorEmail,
```

Comment:
We are not allowed to send this to `authorEmail` (I don't think mg will allow it).  Instead, send it from `support@votebymail.io` as well.  Set `replyTo` to `authorEmail`.